dev/com.ibm.websphere.appserver.features/visibility/public/mpGraphQL-2.0/io.openliberty.mpGraphQL-2.0.feature

```diff
@@ -10,11 +10,11 @@ Subsystem-Name: MicroProfile GraphQL 2.0
 -features= \
   com.ibm.websphere.appserver.internal.slf4j-1.7, \
   com.ibm.websphere.appserver.org.reactivestreams.reactive-streams-1.0, \
-  com.ibm.websphere.appserver.servlet-5.0, \
-  io.openliberty.cdi-3.0, \
-  io.openliberty.jakarta.annotation-2.0, \
-  io.openliberty.jsonb-2.0, \
-  io.openliberty.mpCompatible-5.0, \
+  com.ibm.websphere.appserver.servlet-5.0; ibm.tolerates:="6.0", \
+  io.openliberty.cdi-3.0; ibm.tolerates:="4.0", \
+  io.openliberty.jakarta.annotation-2.0; ibm.tolerates:="2.1", \
+  io.openliberty.jsonb-2.0; ibm.tolerates:="3.0", \
+  io.openliberty.mpCompatible-5.0; ibm.tolerates:="6.0", \
   io.openliberty.mpConfig-3.0, \
   io.openliberty.mpContextPropagation-1.3, \
```

Comment:
Because it depends on mpContextPropagation-1.3 you will need concurrent-2.0 tolerates 3.0